app/src/test/java/edu/nextstep/camp/calculator/MainPresenterTest.kt

```diff
@@ -0,0 +1,87 @@
+package edu.nextstep.camp.calculator
+
+import com.google.common.truth.Truth.assertThat
+import edu.nextstep.camp.calculator.domain.Operator
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+
+class MainPresenterTest {
+    private lateinit var presenter: MainPresenter
+    private lateinit var view: MainContract.View
+
+    @Before
+    fun setUp() {
+        view = mockk()
+        presenter = MainPresenter(view)
+    }
+
+    @Test
+    fun `숫자가 입력되면 수식에 추가되고 변경된 수식을 보여줘야 한다`() {
+        // given
+        val expressionSlot = slot<String>()
+        every { view.showExpression(capture(expressionSlot)) } answers { nothing }
+
+        // when
+        presenter.addToExpression(1)
+
+        // then
+        val actual = expressionSlot.captured
+        assertThat(actual).isEqualTo("1")
+        verify { view.showExpression(actual) }
+    }
+
+    @Test
+    fun `연산자가 입력되면 수식에 추가되고 변경된 수식을 보여줘야 한다`() {
+        // given
+        val expressionSlot = slot<String>()
+        every { view.showExpression(capture(expressionSlot)) } answers { nothing }
```

Comment:
```suggestion
        every { view.showExpression(capture(expressionSlot)) } just Runs
```
이 부분은 이렇게도 작성할 수 있습니다 🙂